databases/migrations/chat-plugins/v6.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN TRANSACTION;
+
+-- Logs of each flag
+CREATE TABLE perspective_flags (
+	userid TEXT NOT NULL,
+	score NUMBER NOT NULL,
+	certainty NUMBER NOT NULL,
+	type TEXT NOT NULL,
+	roomid TEXT NOT NULL,
+	time NUMBER NOT NULL
+);
+
+CREATE INDEX type_idx ON perspective_flags(type);
+CREATE INDEX user_idx ON perspective_flags(userid);
```

Comment:
Wouldn't it be better to index on both of them?

Reply:
Yeah. Done.